_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,389 @@
+
+The text content automatically updates every 3 seconds after the page completes loading. A button is available to pause and resume the automatic updates.
+
+```html
+<body onload="toggle()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="toggle()" value="Pause updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		var updating = false
+
+		function toggle() {
+			var control = document.getElementById('control')
+			if (updating) {
+				control.value = 'Resume updates'
+				updating = false
+				clearInterval(updates)
+			} else {
+				control.value = 'Pause updates'
+				updating = true
+				updates = setInterval(change, 3000)
+			}
+		}
+	</script>
+</body>
+```
+
+#### Passed Example 3
+
+The text content automatically updates every 3 seconds after the page completes loading. A button is available to hide the automatically updating content.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="hide()" value="Hide updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		function start() {
+			setInterval(change, 3000)
+		}
+
+		function hide() {
+			var target = document.getElementById('target')
+			target.style.visibility = 'hidden'
+		}
+	</script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text content automatically updates every 3 seconds after the page completes loading. There is no component to stop or pause the automatic updates.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+	</script>
+</body>
+```
+
+#### Failed Example 2
+
+The text content (part of a progress bar) automatically updates every second after the page completes loading. There is no component for the user to pause the auto-updates, other than waiting for them to end.
+
+```html
+<html>
+	<head>
+		<title>Example page for rule efbfc7</title>
```

Comment:
I'm not sure why only this example has a `title` (which is not really needed for the rule).

Reply:
It is the only one with `html` and `head` elements. The tests require a `title` when you have an `html` element, but not otherwise.